pkgs/development/python-modules/augmax/default.nix

```diff
@@ -1,16 +1,19 @@
-{ buildPythonPackage
+{ lib
+, buildPythonPackage
 , einops
 , fetchFromGitHub
 , jax
 , jaxlib
-, lib
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "augmax";
   version = "unstable-2022-02-19";
   format = "setuptools";
 
+  disabled = pythonOlder "3.7";
```

Comment:
why is this necessary? I don't think we have anything older than python3.7 packaged in nixpkgs?